adapter/internal/oasparser/model/types.go

```diff
@@ -151,9 +155,36 @@ type APIYaml struct {
 			SandboxFailoverEndpoints     []EndpointInfo `json:"sandbox_failovers,omitempty"`
 			ImplementationStatus         string         `json:"implementation_status,omitempty"`
 		} `json:"endpointConfig,omitempty"`
+		Operations []OperationYaml `json:"Operations,omitempty"`
 	} `json:"data"`
 }
 
+// OperationYaml holds attributes of APIM operations
+type OperationYaml struct {
+	ID              string `json:"id,omitempty"`
+	Target          string `json:"target,omitempty"`
+	Verb            string `json:"verb,omitempty"`
+	RateLimitPolicy string `json:"rateLimitPolicy,omitempty"`
+}
+
+// APIRateLimitPolicy holds policy details relevant to the rate limiting
+type APIRateLimitPolicy struct {
+	PolicyName string `json:"policyName,omitempty"`
+	Type       string `json:"type,omitempty"`
+	Count      int    `json:"count,omitempty"`
+	Unit       string `json:"unit,omitempty"`
+	Span       uint32 `json:"span,omitempty"`
+	SpanUnit   string `json:"spanUnit,omitempty"`
+}
+
+// RateLimitPolicy contains all the fields in the rate-limit_policies.yaml file
+type RateLimitPolicy struct {
+	ApimMeta
```

Comment:
What is this field

Reply:
Similar to the api.yaml
<img width="419" alt="image" src="https://user-images.githubusercontent.com/40932779/206634916-321994f2-9e0a-4dbc-9ecf-f91aff75fc33.png">